### PR TITLE
How To Harden SSH On Ubuntu

### PR DESCRIPTION
**Pull request by NoahDoah**
I have read and understood the Contributor's Certificate of Origin
available at the end of https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Noah Krasser <noah@krasser.xyz>

### CODE REVIEW

File: tutorials/harden-ssh-ubuntu/01.en.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/tutorial-template"
+slug: "tutorial-template"
+date: "2030-01-01"
```
[the remark this answers is not in the document]
Comment (n-se) [reply tutorials/harden-ssh-ubuntu/01.en.md]: Don't forget to edit the `slug` and `path` too.

**Comment by n-se** [comment]
I am just a friendly reminder :)

Your tutorial already looks good but I am afraid that you still have not "harden't" your SSH server. I think if you add some more information about key based authentication and other tricks to harden the SSH server we can get this merged and you get your credit!

**Comment by BackInBash** [comment]
Can i take that on over. It's been a month since the last response. I would add the subjects, key based and two factor authentication.

**Comment by n-se** [comment]
I think so, yes. Just open a new pull request about a new "Harden SSH" guide. If possible don't name it "on <Distribution>", because SSH hardening is mostly the same on all Distributions.

**Comment by n-se** [comment]
Closing in favor of #115

**Pull request closed without merging**